### PR TITLE
Let modules wait until requested stylesheets loaded.

### PR DESCRIPTION
**Pull request by omo**
This is the first part of #262.

### CODE REVIEW

File: spec/modules/custom_spec.js [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
   var Custom = webfont.modules.Custom,
-      FontFamily = webfont.FontFamily;
+      FontFamily = webfont.FontFamily,
+      Any = jasmine.Matchers.Any;
 
   describe('insert links correctly', function () {
     var fakeDomHelper = null,
         load = null;
 
+    function notiySheetsLoaded() {
```
Comment (bramstein) [inline spec/modules/custom_spec.js]: Typo here

File: src/core/domhelper.js [hunk 2]
```diff
@@ -23,6 +23,17 @@ goog.scope(function () {
   var DomHelper = webfont.DomHelper;
 
   /**
+   * The NativeFontWatchRunnner depends on the correct and reliable
+   * |onload| event, and browsers with the native font loading API
+   * have reliable @onload support as far as we know. So we use the
+   * event for such a case and unconditionally invokes the callback
+   * otherwise.
+   *
+   * @const
```
Comment (bramstein) [inline src/core/domhelper.js]: This should have a `@type {boolean}`

File: src/core/domhelper.js [hunk 3]
```diff
@@ -23,6 +23,17 @@ goog.scope(function () {
...
+   */
+  DomHelper.canWaitStylesheet = !!window['FontFace'];
```
Comment (bramstein) [inline src/core/domhelper.js]: Constants should be in uppercase with underscores (e.g. `CAN_WAIT_STYLESHEET`).

**Comment by omo** [comment]
@bramstein Thanks for the review! Updated. Could you take another look?

File: src/core/domhelper.js [hunk 4]
```diff
@@ -23,6 +23,17 @@ goog.scope(function () {
   var DomHelper = webfont.DomHelper;
 
   /**
+   * The NativeFontWatchRunnner depends on the correct and reliable
+   * |onload| event, and browsers with the native font loading API
+   * have reliable @onload support as far as we know. So we use the
+   * event for such a case and unconditionally invokes the callback
+   * otherwise.
+   *
+   * @const @type{boolean}
```
Comment (bramstein) [inline src/core/domhelper.js]: The `@type {boolean}` should be on a separate line.

**Comment by bramstein** [comment]
One small thing, otherwise I think we're good to ship this.

File: src/core/domhelper.js (hunk 2 above)
Comment (omo) [inline src/core/domhelper.js]: Fixed.

**Comment by omo** [comment]
@bramstein Thanks for the review! I updated the branch with the fix.
Let's have this and #275 in !

**Pull request merged**